codegen_java/build.sbt

```diff
@@ -6,22 +6,26 @@ lazy val root = (project in file(".")).
     Seq(organization := "org.broadinstitute.cromwell",
     name := "cromwell-client",
     version := createVersion("0.1"),
-    scalaVersion := "2.12.15",
+    scalaVersion := "2.13.8",
     scalacOptions ++= Seq("-feature"),
     compile / javacOptions ++= Seq("-Xlint:deprecation"),
     Compile / packageDoc / publishArtifact := false,
     resolvers += Resolver.mavenLocal,
     updateOptions := updateOptions.value.withGigahorse(false),
     libraryDependencies ++= Seq(
-      "io.swagger" % "swagger-annotations" % "1.5.21",
-      "com.squareup.okhttp3" % "okhttp" % "3.12.1",
-      "com.squareup.okhttp3" % "logging-interceptor" % "3.12.1",
-      "com.google.code.gson" % "gson" % "2.8.5",
+      "io.swagger" % "swagger-annotations" % "1.6.5",
+      "com.squareup.okhttp3" % "okhttp" % "4.9.3",
+      "com.squareup.okhttp3" % "logging-interceptor" % "4.9.3",
+      "com.google.code.gson" % "gson" % "2.9.0",
       "org.apache.commons" % "commons-lang3" % "3.12.0",
       "org.apache.oltu.oauth2" % "org.apache.oltu.oauth2.client" % "1.0.1",
-      "org.threeten" % "threetenbp" % "1.3.5" % "compile",
-      "io.gsonfire" % "gson-fire" % "1.8.0" % "compile",
-      "junit" % "junit" % "4.12" % "test",
-      "com.novocode" % "junit-interface" % "0.10" % "test"
+      "javax.ws.rs" % "javax.ws.rs-api" % "2.1.1",
+      "javax.annotation" % "javax.annotation-api" % "1.3.2",
+      "com.google.code.findbugs" % "jsr305" % "3.0.2",
+      "org.threeten" % "threetenbp" % "1.6.0" % Compile,
+      "io.gsonfire" % "gson-fire" % "1.8.5" % Compile,
+      "junit" % "junit" % "4.13.2" % Test,
+      "com.novocode" % "junit-interface" % "0.11" % Test,
+      "org.junit.jupiter" % "junit-jupiter-api" % "5.8.2" % Test
```

Comment:
turns out our codegen was broken (BT-48) and when I fixed it I found a bunch of dependencies that needed to be upgraded or added

Reply:
Good news, bad news:

The good news is I think Rawls should be able to work with the existing `cromwell-client` now and into the Rawls 2.13 future. Because the `cromwell-client` artifact is comprised entirely of Java files (generated from OpenAPI YAML) it can be treated like a regular Java artifact. The jar for `cromwell-client` is named with a Scala version because it is being produced by this `build.sbt`, but there's nothing actually Scala-version-dependent within the jar. [Rawls PR](https://github.com/broadinstitute/rawls/pull/1680/files) to make this more clear in the Rawls `build.sbt`.

The bad news is that while the changes I made to this `build.sbt` do seem to be an improvement from the current completely broken state of API codegen in Cromwell, the artifact built with these changes seems to cause issues in two in Rawls CI tests (`DataRepoEntityProviderSpec` and `SubmissionSpec`). Because of the "good news" item above this is not a concern for the foreseeable future, but if at some point an actual API change is required this situation would need to be addressed.

Reply:
> cause issues in two in Rawls CI tests (DataRepoEntityProviderSpec and SubmissionSpec

Tested seem all passed in the rawls PR...is there sth else indicating those tests are broken?

Reply:
Interesting result! Yes these failed multiple times in my Rawls PRs yesterday against develop and yesterday's version of your 2.13 branch. I'll have another look now.

Reply:
tl;dr I stand by my comment above that this does not completely fix the broken `cromwell-client` building described in BT-38, but it's certainly no worse than the status quo. 

I confirmed the Rawls unit tests are still failing with a branch based on Rawls develop that uses the `cromwell-client` artifact built by this `build.sbt`:

```
11:43:54 [info] *** 2 TESTS FAILED ***
11:43:54 [error] Failed tests:
11:43:54 [error] 	org.broadinstitute.dsde.rawls.entities.datarepo.DataRepoEntityProviderSpec
11:43:54 [error] 	org.broadinstitute.dsde.rawls.jobexec.SubmissionSpec
11:43:55 [error] (rawlsCore / Test / test) sbt.TestsFailedException: Tests unsuccessful
```

With a branch of Qi's 2.13 branch that uses this artifact the unit tests pass but not the `rawls-fiab-test-runner`:

```
[info] Tests: succeeded 79, failed 2, canceled 0, ignored 0, pending 0
[info] *** 2 TESTS FAILED ***
[error] Failed tests:
[error] 	org.broadinstitute.dsde.test.api.WorkspaceApiSpec
[error] (Test / testOnly) sbt.TestsFailedException: Tests unsuccessful
```